paging-runtime-composeui/src/androidMain/kotlin/app/cash/paging/AndroidLogger.kt

```diff
@@ -0,0 +1,28 @@
+package app.cash.paging
+
+import android.util.Log
+import androidx.paging.LOG_TAG
+import androidx.paging.Logger
+
+internal actual fun createLogger(): Logger {
+  return object : Logger {
+    override fun isLoggable(level: Int): Boolean {
+      return Log.isLoggable(LOG_TAG, level)
+    }
+
+    override fun log(level: Int, message: String, tr: Throwable?) {
+      when {
+        tr != null && level == Log.DEBUG -> Log.d(LOG_TAG, message, tr)
+        tr != null && level == Log.VERBOSE -> Log.v(LOG_TAG, message, tr)
+        level == Log.DEBUG -> Log.d(LOG_TAG, message)
+        level == Log.VERBOSE -> Log.v(LOG_TAG, message)
```

Comment:
There is no need to do this split codepath. All logging methods accept a nullable `Throwable`.

Reply:
Agreed, just copied this over from upstream. I'll consolidate

Reply:
There's no named argument usage here, though. The `Log.d` and `Log.v` methods in Java accept a nullable `Throwable`, and the ones which do not take a `Throwable` are explicitly passing `null` as the throwable anyway so having a check here is basically redundant.

Reply:
Sent a CL to fix upstream: http://r.android.com/2363040

Reply:
Nice ty